### ISSUE TITLE
Improving the Web Widget

### ISSUE THREAD

**Issue by grillol** [botman/web-widget#24]
Hi Marcel, thanks for the job you have done. 

I would like to improve the web widget action.

For example, I would like the user to fill in a list of values before sending the message (ex: work experience list), or to have an autocomplete option, etc.

Would it be possible to have minimal documentation of the code and some suggestions to contribute on this direction? 

Thanks a lot.

**Comment by danlopez00** [comment]
+1

**Comment by Dontorpedo** [comment]
+1

**Comment by arch2603** [comment]
I am interested to know if there are documentations available that will allow us to customize the widget, please?

**Comment by Dontorpedo** [comment]
absoulutely +1

**Comment by teevon** [comment]
absolutely, I am currently experiencing issues trying to turn off the default autocomplete setting of the #userText input field (The #userText input field is the web widget input field, I inspected and found out the id attribute was userText)

**Comment by adelmazza** [comment]
> absolutely, I am currently experiencing issues trying to turn off the default autocomplete setting of the #userText input field (The #userText input field is the web widget input field, I inspected and found out the id attribute was userText)

You can add this after chat.js:

`<script>
    window.addEventListener('load', function () {
        document.getElementById("userText").setAttribute("autocomplete","off");
    })
</script>`

**Comment by teevon** [comment]
> <script> window.addEventListener('load', function () { document.getElementById("userText").setAttribute("autocomplete","off"); }) </script>

I went to the chat.js and directly added set the autocomplete attribute to off, but I prefer your solution better, makes it more flexible for future changes, if the need arises, updated my code, thank you.

**Comment by teevon** [comment]
> > absolutely, I am currently experiencing issues trying to turn off the default autocomplete setting of the #userText input field (The #userText input field is the web widget input field, I inspected and found out the id attribute was userText)
> 
> You can add this after chat.js:
> 
> `<script> window.addEventListener('load', function () { document.getElementById("userText").setAttribute("autocomplete","off"); }) </script>`

I went to the chat.js and directly added set the autocomplete attribute to off, but I prefer your solution better, makes it more flexible for future changes, if the need arises, updated my code, thank you.

**Comment by teevon** [comment]
> > absolutely, I am currently experiencing issues trying to turn off the default autocomplete setting of the #userText input field (The #userText input field is the web widget input field, I inspected and found out the id attribute was userText)
> 
> You can add this after chat.js:
> 
> `<script> window.addEventListener('load', function () { document.getElementById("userText").setAttribute("autocomplete","off"); }) </script>`

@adelmazza I went to the chat.js and directly added set the autocomplete attribute to off, but I prefer your solution better, makes it more flexible for future changes, if the need arises, updated my code, thank you.

**Comment by EranGrin** [comment]
I created a new package that solves this and several other issues
https://www.npmjs.com/package/botman-extended-web-widget
